src/test/uint256_tests.cpp

```diff
@@ -6,12 +6,15 @@
 #include <streams.h>
 #include <test/util/setup_common.h>
 #include <uint256.h>
+#include <util/strencodings.h>
+#include <util/transaction_identifier.h>
 
 #include <boost/test/unit_test.hpp>
 
 #include <iomanip>
 #include <sstream>
 #include <string>
+#include <string_view>
```

Comment:
iwyu reports this include as unneeded, but it is obviously used: https://cirrus-ci.com/task/6533090686795776?logs=ci#L17964